packages/data-context/src/DataContext.ts

```diff
@@ -141,6 +157,11 @@ export class DataContext {
     return new FileDataSource(this)
   }
 
+  @cached
+  get node () {
```

Comment:
This seems like a kind of bad name, considering we also use `node` as in `node.js` quite a bit in the code base. Maybe `GraphQLNodeDataSource`? 🤔 naming is hard.

Reply:
Moving it to the `GraphQLDataSource` - `ctx.graphql.resolveNode`